### PR TITLE
Donate page

### PR DESCRIPTION
**Pull request by Vivsss**

### CODE REVIEW

File: content/donate/_index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: "Donate now to support internet freedom. As a non-profit, we rely on donations from people like you to defend a free and open internet for all."
+maintext: "Donate now to support internet freedom"
+maindescr: "As a non-profit, we rely on donations from people like you to defend a free and open internet for all. By donating to OONI, you support:"
+why2supports:
+- text: "Resurgence of Internet Censorship in Ethiopia: Blocking of WhatsApp, Facebook, and African Arguments"
+  link: "https://ooni.org/post/resurgence-internet-censorship-ethiopia-2019/"
+- text: "Ethiopia: Verifying the unblocking of websites"
+  link: "https://ooni.org/post/ethiopia-unblocking/"
+- text: "#EgyptCensors: Evidence of recent censorship events in Egypt"
+  link: "https://ooni.org/post/egypt-censors/"
```
Comment (hellais) [inline content/donate/_index.md:12]: These links should not be here

File: content/donate/_index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+* [Free and open source software](https://ooni.org/install/) for detecting internet censorship
+
+* [Real-time analysis](https://github.com/ooni/pipeline/) of internet censorship around the world
+
+* [Open data](https://ooni.org/data/) on internet censorship -- supporting global research and advocacy efforts
+
+* [Research](https://ooni.org/reports/) on censorship events worldwide
```
Comment (hellais) [inline content/donate/_index.md:21]: This content should not be present here.

File: content/donate/stickers/index.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+name: "OONI stickers"
+isEnabled: false
+---
```
Comment (hellais) [inline content/donate/stickers/index.md:4]: Are these still necessary?

File: layouts/donate/single.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <ul>
+    {{ range .Params.reports }}
+      <li><a href="{{ .link }}">{{ .text }}</a></li>
+    {{ end }}
+    </ul>
+
+  <a class="back-link" href="/partners/">← back to partners</a>
+
+</div>
+{{ end }}
```
Comment (hellais) [inline layouts/donate/single.html:38]: I don't think this page is necessary.

File: layouts/section/donate.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      </div>
+    
+   <div class="container">   
+    <div class="row">   
+       <div class="col-12 details">
+       <h3>Thank you for supporting the OONI-verse!</h3>
+       <p><bold>Donate now to support the global fight against internet censorship!</bold></p>
+       
+       <p>{{ .Params.Maindescr }}</p>
+        {{ .Content }}
```
Comment (hellais) [inline layouts/section/donate.html:18]: Can we put all the content inside of the markdown file so it's easier to edit and translate?

File: content/donate/_index.md (hunk 2 above)
Comment (hellais) [reply content/donate/_index.md:21]: I now realise this is needed, but should include also the content that is currently inlined in the HTML.

File: layouts/section/donate.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+ 
+{{ define "script" }}
+  <script type="text/javascript" src="/js/zepto.min.js"></script>
+<script src="https://www.paypal.com/sdk/js?client-id=ATd3rHkGu0i2Q1KsJpk3at-MR6pFfBIYgrJuDFsxVau4_lS_KEhydMUIu0lWKCcPwNvc9Fmyp6JUCV-Q&vault=true"  data-namespace="paypal_sdk"></script>
+  <script type="text/javascript">
+var myAmount;
+var recurrency; 
+var payment;
+  $(document).ready(function(){ 
+
```
Comment (hellais) [inline layouts/section/donate.html:46]: This is a left-over broken javascript code, which needs to be removed.

**Review by hellais — CHANGES_REQUESTED** [review]
Should remove all the dead code and ensure that all the copy/content of the donate page is included inside of the `content/donate/_index.md`

File: content/donate/_index.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Donate"
+description: "Donate now to support internet freedom. As a non-profit, we rely on donations from people like you to defend a free and open internet for all."
+maintext: "Donate now to support internet freedom"
+maindescr: "As a non-profit, we rely on donations from people like you to defend a free and open internet for all. By donating to OONI, you support:"
```
Comment (hellais) [inline content/donate/_index.md:5]: Can we move this into the content?

File: layouts/section/donate.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+       <h3>Thank you for supporting the OONI-verse!</h3>
+       <p><bold>Donate now to support the global fight against internet censorship!</bold></p>
+       
+       <p>{{ .Params.Maindescr }}</p>
+        {{ .Content }}
+       
+        <div class="row col-12">
+            <h4 class="col-12">Other ways to give:</h4>
+            <ul>
+                <li>Bank Transfer / SEPA</li>
```
Comment (hellais) [inline layouts/section/donate.html:23]: Missing link and page for this

File: layouts/section/donate.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+  </div>
+   </div>
+
+</div>
+
+{{ end }}
+ 
+{{ define "script" }}
+  <script type="text/javascript" src="/js/zepto.min.js"></script>
+<script src="https://www.paypal.com/sdk/js?client-id=ATd3rHkGu0i2Q1KsJpk3at-MR6pFfBIYgrJuDFsxVau4_lS_KEhydMUIu0lWKCcPwNvc9Fmyp6JUCV-Q&vault=true"  data-namespace="paypal_sdk"></script>
```
Comment (hellais) [inline layouts/section/donate.html:40]: We don't need any javascript since we are using a more simple button

File: layouts/section/donate.html [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+        <div class="h1-container">
+            <h1>{{ .Params.Maintext }}</h1>
+            <a id='donate' href={{ .Params.PaypalButton }} class="btn btn-outline-primary col-6">{{.Params.paypalbuttonLabel}}</a> 
+        </div>
+      </div>
+    
+   <div class="container">   
+        <div class="row">   
+       <div class="col-12 details">
+           <h3>{{ .Params.Thanks }}</h3>
```
Comment (hellais) [inline layouts/section/donate.html:14]: This parameter can go in the markdown directly

File: content/donate/_index.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Donate"
+maintext: "Donate now to support internet freedom"
+thanks: "Thank you for supporting the OONI-verse!"
```
Comment (hellais) [inline content/donate/_index.md:4]: This can go inside of the body

**Review by agrabeli — CHANGES_REQUESTED** [review]
I've reviewed and edited the copy. The cancel page though needs to be deleted since we're removing the paypal option.

**Pull request merged**